doc/release-notes.md

```diff
@@ -32,61 +36,190 @@ PIVX Core should also work on most other Unix-like systems but is not frequently
+ - #1801 `30d353edab` [Core] Per-txout model for chainstate database (random-zebra)
+ - #1804 `5c8b992033` [Core] Use std::unordered_{map,set} (C++11) instead of boost::unordered_* (random-zebra)
 
-### Build System
+### GUI
+ - #1754 `93d574170d` [Model] Wallet interface refactor + code cleanup. (furszy)
+ - #1776 `2ad27b1407` [Model] TransactionRecord decomposeTransaction refactoring (furszy)
+ - #1782 `ada4462782` [GUI] Start masternodes polling timer only when is needed. (furszy)
+ - #1805 `f0cc6fcc38` [BUG][GUI] Don't append cold-stake records twice (random-zebra)
+ - #1863 `ad15bce2f5` [Trivial][GUI] Fix init messages (random-zebra)
 
-### P2P Protocol and Network Code
+### Wallet
+ - #1752 `2e32285a70` [Wallet] Simple unused methods cleanup. (furszy)
+ - #1755 `eeb129b477` [wallet] List COutput solvability + wallet_ismine refactoring. (furszy)
+ - #1757 `e2cc4aa411` [Wallet] add cacheable amounts for caching credit/debit values (furszy)
+ - #1759 `dcc92f8157` [Wallet] AvailableCoins remove duplicated watchonly config argument. (furszy)
+ - #1760 `3b030f9978` [Wallet] AvailableCoins code readability improved (furszy)
+ - #1764 `6847d0d648` [Wallet] Securely erase potentially sensitive keys/values (Thomas Snider)
+ - #1767 `8ab63d3e5b` [Wallet] Ignore MarkConflict if block hash is not known (random-zebra)
+ - #1781 `4715915d4c` [Wallet] Acquire cs_main lock before cs_wallet during wallet initialization (random-zebra)
+ - #1783 `abf7c62934` [Wallet] Do not try to resend transactions if the node is importing or in IBD (furszy)
+ - #1787 `4b1f3eb792` [Wallet] Improve usage of fee estimation code (random-zebra)
+ - #1802 `7db7724cff` [Wallet] Make nWalletDBUpdated atomic to avoid a potential race (furszy)
+ - #1810 `49bd99929d` [Wallet] wtx cached balances test coverage + getAvailableCredit problem fix. (furszy)
+ - #1811 `e89e20eca1` [Wallet][Refactoring] wallet/init refactoring backports (random-zebra)
+ - #1817 `6480c7d9bf` [Wallet] Speedup coinstake creation removing redundancies. (furszy)
+ - #1832 `c14d130b48` [Wallet] Cleanup getbalance methods that are not fulfilling any purpose. (furszy)
 
-### GUI
+### P2P Protocol and Network Code
+ - #1769 `1e334200bb` [Net] Remove bogus assert on number of oubound connections. (Matt Corallo)
+ - #1780 `5fcad0c139` [Net] cs_vSend-cs_main deadlock detection fixed. (furszy)
+ - #1800 `616b102f8b` [P2P] Improve AlreadyHave (Alex Morcos)
+ - #1812 `777638e7bc` [P2P] Begin Network Encapsulation (random-zebra)
+ - #1835 `cbd9271afb` [Net] Massive network refactoring and speedup (Fuzzbawls)
 
 ### RPC/REST
-
-### Wallet
+ - #1753 `e288a4508b` [Trivial] [RPC] Fix listcoldutxos help text (JSKitty)
+ - #1828 `4fc36b59ee` [RPC][BUG] Fix ActivateBestChain calls in reconsider(invalidate)block (random-zebra)
+ - #1831 `28509bf9e8` [RPC] re introducing filtering args in getbalance (furszy)
+
+### Build Systems
+ - #1553 `fddf765132` [Build] Sapling Foundations (Build System + ZIP32) (Fuzzbawls)
```

Comment:
(furszy)

Reply:
fixed...the script that generates this list doesn't handle multi-author PRs very well :P